site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md

```diff
@@ -0,0 +1,108 @@
+
+There are two ways for Contour to find this information:
+- by watching a Service object for the Envoy service, and putting the associated `status.loadBalancer` block from that Service into all associated Ingress objects. This is what is used in the example deployment.
+- Operators can also specify an address on Contour's command line, using the `--ingress-status-address` flag. The address that’s passed on the command line will be passed straight through to the Ingress status.
+
+This also means that when you `kubectl get` a Contour-owned Ingress, instead of this:
+
+```
+$ kubectl get ingress httpbin
+NAME      HOSTS                   ADDRESS   PORTS     AGE
+httpbin   httpbin.youngnick.dev             80, 443   336d
+```
+you will see this:
+
+```
+$ kubectl get ingress httpbin
+NAME      HOSTS                   ADDRESS   PORTS     AGE
+httpbin   httpbin.youngnick.dev   x.x.x.x   80, 443   336d
+
+```
+
+### Removed the `--use-extensions-v1beta1-ingress` flag
+
+The `--use-extensions-v1beta1-ingress` flag was removed from the contour serve command in Contour 1.3. If you have a previous deployment that specifies this command, you must remove it or Contour will fail to start.
+
+## Future Plans
+
+The Contour project is very community-driven and the team would love to hear your feedback! 
+
+- Come talk about topics at our next community meeting.
+- We’ve heard that a number of teams have forked Contour and we would love to hear about what changes you needed, and to see if we can help to bring them upstream.
+Please consider coming to our community meeting, or contact us: either via an issue, or hit me up on Twitter [@youngnick](https://twitter.com/youngnick).
+
+If you are interested in contributing, a great place to start is to comment on one of the issues labeled with [Help Wanted]({{site.github.repository_url}}/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22Help+wanted%22+) and work with the team on how to resolve them. 
+
+## Are you a Contour user? We would love to know!
+If you're using Contour and want to add your organization to our adopters list, please visit this [page](https://github.com/projectcontour/contour/blob/master/ADOPTERS.md).
+If you prefer to keep your organization name anonymous but still give us feedback into your usage and scenarios for Contour, please post on this [GitHub thread](https://github.com/projectcontour/contour/issues/1269)          
+
+## Join the Contour Community!
+- Join the Contour Community Meetings via the instructions on our [Community Page](https://projectcontour.io/community/)
+- Get updates on Twitter ({{site.footer_social_links.Twitter.url}})
+- Chat with us on Slack ({{site.footer_social_links.Slack.url}})
+- Collaborate with us on GitHub: {{site.github.repository_url}}
```

Comment:
This is covered in a community footer for each blog so you don't have to add it: https://github.com/projectcontour/contour/blob/master/site/_includes/blog-community-footer.html